### ISSUE TITLE
Default action is not correct

### ISSUE THREAD

**Issue by brianreidc7** [MicrosoftDocs/microsoft-365-docs#10974]
The default for the common attachment filter is to reject and not to quarantine as this page says. Quarantine is a new feature here, but it is not the default setting.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: db041b73-f7c9-db6c-070b-a7c974de640c
* Version Independent ID: dffb7045-3ba8-a014-d1be-462ca1d609b5
* Content: [Configure anti-malware policies - Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/anti-malware-policies-configure?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/anti-malware-policies-configure.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/anti-malware-policies-configure.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@brianreidc7, tested and confirmed. I updated the affected content. This was apparently announced in MC468187.

Whenever there's talk of a "default value", the following locations are in play for any security policy:

- The default value in new policies that you create (with possible differences between created in the UI vs. created in PowerShell).
- The default value in the default policy.
- The default value in existing custom policies (likely untouched).
- The default value in Standard and Strict preset security policies.

I checked them all in a recent Microsoft 365 E5 tenant, and the default value was indeed Reject for the FileTypeAction setting on anti-malware policies.

The common attachments filter itself is on by default in new anti-malware policies created in the UI, but off by default in the default anti-malware policy and new policies created in PowerShell.
